tests/test_tool_loader.py

```
@@ -325,12 +317,11 @@ def test_os_env_key_injected_into_server_config(self):
             },
         ):
-            from src.chanakya.services import tool_loader
-
             # Patch 'os' into the module's namespace to fix the missing import
-            import importlib
             import os as _os
 
+            from src.chanakya.services import tool_loader
+
             tool_loader_module = sys.modules["src.chanakya.services.tool_loader"]
             setattr(tool_loader_module, "os", _os)
```

Ruff (0.15.6)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)